.github/workflows/drip-django.yaml

```diff
@@ -0,0 +1,64 @@
+name: Django Drip Campaigns
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+        # python 3.4
```

Comment:
Are these comments adding value?

Reply:
Yes, I think it could be useful in the future if there is a need of changing Django versions (or any other library) and keeping consistence in grouping by Python versions.  WDYT?